en/organization_management/daily_management.md

```diff
@@ -0,0 +1,61 @@
+# Daily Management
+
+**NOTE**: The following operations can only be performed by organization administrators.
+
+After organization administrator login to Seafile, he/she can go to organization admin page by click **Organization Admin**
+
+![](../images/auto-upload/WX20230615-180101@2x.png)
+
+## View System Info
+
+In organization admin page, administrator can view the system usage, graphical statistics of this organization.
+
+![](../images/auto-upload/WX20230616-102525@2x.png)
+![](../images/auto-upload/WX20230616-102743@2x.png)
+
+## View Libraries and Operation Logs
+
+Administrators can not only view all libraries in this organization, but also can view some operation logs, such as file access/update and permission log.
+
+![](../images/auto-upload/WX20230616-105147@2x.png)
+![](../images/auto-upload/WX20230616-115301@2x.png)
+
+## Manage Devices
+
+Devices in organization admin page can list all the devices connected to this organization, administrator can unlink a certain device and view the detailed device connection error message.
```

Comment:
All connected devices from the users in the organization can be managed in this page. You can view access information of the devices, unlink devices. You can also check sync errors for desktop clients.